packages/http-client-java/generator/http-client-generator-test/Generate.ps1

```diff
@@ -57,6 +57,10 @@ $generateScript = {
     $tspOptions += " --option ""@typespec/http-client-java.api-version=2022-09-01"""
     # exclude preview from service versions
     $tspOptions += " --option ""@typespec/http-client-java.service-version-exclude-preview=true"""
+  } elseif ($tspFile -match "type[\\/]array" -or $tspFile -match "type[\\/]dictionary") {
+    # TODO https://github.com/Azure/autorest.java/issues/2964
```

Comment:
Does released services have this case?  If they have, do we need to hold on the releases on typespec-java until the issue fixed?

Reply:
We've done a test sync, no released lib has it. -- this is pretty rare for service to have body as raw array or dict, and rarer for the element to be unknown.